_posts/2016-10-31-recurse-center-week-5.md

```diff
@@ -0,0 +1,172 @@
+---
+layout: post
+title: "Week[4] @ The Recurse Center 😳"
+comments: true
+categories: tech recurse
+---
+
+Almost half way through. Things getting serious!
+
+# Overall
+
+Last week started so good, but by the end I was re-evaluating my goals... allow me to explain...
+
+### Emotional 😫
+
+#### **Saying no more**
+
+When I first got to RC, my strategy was to say yes to everything.
+
+> "Want to work together? Yes!"
+
+> "Want to go to X? Yes!"
+
+> "Want to learn about <insert obscure topic>? Yes!"
+
+I love when others are excited about things.  
+I love getting caught up in their excitement and encouraging it.
+
+But it can also be very distracting from what I'm excited about.
+
+Sometimes it feels like saying no to an invitation will leave me with nothing to do. I forget all the things I'm excited about doing: improv, dance classes, parks, climbing, reading, etc., etc.
+
```

Comment:
You should read [this book](https://www.amazon.com/Essentialism-Disciplined-Pursuit-Greg-McKeown/dp/0804137382). You may be able to find the ebook version for free but either way it really made "saying no" to things easier for me.